searx/engines/mozhi.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+"""Mozhi (alternative frontend for popular translation engines)"""
+
+import random
+import re
+from urllib.parse import urlencode
+
+about = {
+    "website": 'https://codeberg.org/aryak/mozhi',
+    "wikidata_id": None,
+    "official_api_documentation": 'https://mozhi.aryak.me/api/swagger/index.html',
+    "use_official_api": True,
+    "require_api_key": False,
+    "results": 'JSON',
+}
+
+engine_type = 'online_dictionary'
+categories = ['general']
+
+base_url = "https://mozhi.aryak.me"
+mozhi_engine = "yandex"
```

Comment:
Any reason why you use `yandex` by default? .. for me `yandex` had very slow response and  `deepl`, `goggle` and `mymemory` have good translations and do fast response.

`reverso` did not worked for me / I tested with:

```
!mozhi en-de SearXNG is a free internet metasearch engine which aggregates results from more than 70 search services. Users are neither tracked nor profiled. Additionally, SearXNG can be used over Tor for online anonymity.
```

Reply:
The only reason is that Yandex suppports additional example sentences for a translation (which I needed for testing). I haven't tested the other supported engines really, there probably is a better alternative, we can change it to whatever translation engine you prefer :)

Reply:
I vote for google / afaik it has the most translations (languages).